tests/api/v1alpha2/tests.dhall

```diff
@@ -143,8 +143,19 @@ in  { accessSpec =
       }
     , networkingSpec =
       { amazonVPC =
-          Kops.NetworkingSpec.AmazonVPC
-            { amazonvpc = Kops.Networking.AmazonVPC::{=} }
+        { default =
+            Kops.NetworkingSpec.AmazonVPC
+              { amazonvpc = Kops.Networking.AmazonVPC::{=} }
+        , with-env =
+            Kops.NetworkingSpec.AmazonVPC
+              { amazonvpc = Kops.Networking.AmazonVPC::{
+                , env = Some
+                  [ { name = "WARM_IP_TARGET", value = "10" }
+                  , { name = "AWS_VPC_K8S_CNI_LOGLEVEL", value = "debug" }
```

Comment:
Should set it to:
`[ { name = "WARM_IP_TARGET", value = "6" }
     , { name =  "MINIMUM_IP_TARGET", value = "3" }
]`

Reply:
Sorry for that!